src/components/grid/grid.ts

```diff
@@ -170,6 +172,15 @@ export class Grid extends Component<Props, SpreadsheetChildEnv> {
       this.hoveredCell.clear();
     });
     this.cellPopovers = useStore(CellPopoverStore);
+
+    useEffect(
+      () => {
+        if (!this.sidePanel.isOpen) {
+          this.focus();
```

Comment:
there's only one use of `focusGrid` in `Spreadsheet` component. Getting rid of this last use is my next step.

Reply:
Pretty sure it's not introduces by this PR bus since some side panels (pivot properties/list/ chart) close themselves up following a collaborative command:  If I'm currently editing a link for instance - the sidepanel closing will make me lose focus of my input box :'(